frontend/wallet/src/main/Application.re

```diff
@@ -1,39 +1,15 @@
-open Tc;
-
 module Action = {
```

Comment:
this will destroy me in conflicts, but oh well

Reply:
I didn't do anything important in this file, feel free to remove any changes I made as long as the end product compiles